plugins/woocommerce/src/Admin/API/Options.php

```diff
@@ -219,6 +219,7 @@ public static function get_default_option_permissions() {
 			'woocommerce_admin_customize_store_completed_theme_id',
 			'woocommerce_admin_customize_store_survey_completed',
 			'woocommerce_coming_soon',
+			'woocommerce_coming_soon_hide_admin_bar_badge',
```

Comment:
Looks like we don't use Option API to get or update the `woocommerce_coming_soon_hide_admin_bar_badge` option. Is this required?

Reply:
Don't think so, thanks for catching it! I put it in because initially it didn't save properly as I didn't add it to the list in LaunchYourStore.php.